### PR TITLE
two_ais: persist scene objects via dataclass-csv

### PR DESCRIPTION
**Pull request by emberian**
a "CSV adapter" for not having to edit `main.py`
Closes HCX-61

### CODE REVIEW

File: example_scene.csv
Comment (emberian) [inline example_scene.csv:1]: love this scene! don't commit generated files though...

- [x] can you add `example_scene.csv` to the `.gitignore`?
- [x] make sure to delete this file before you make the commit that includes the change to the `.gitignore`!

File: generate_example_csvs.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        id=random.randint(0, 10),
+        name="Avatar 3: The Search for More Money",
+        description="James Camerons' newest hit single, Avatar 3, unrelated in any way to the previous instantiations because GPT-4 has no persistence or mutability!",
+        characters=[cast.Avatar, cast.Avatar2, cast.KillerOfWorlds],
+        text_only=True,
+    )
+
+    with open("example_scene.csv", "w") as f:
+        w = DataclassWriter(f, [example_scene], scene.Scene)
+        w.write()
```
Comment (emberian) [inline generate_example_csvs.py:27]: maybe it'd be cool to call out to `main.py --two_ais --scene=example_scene.csv` or something?

File: main.py [hunk 2]
```diff
@@ -137,12 +138,15 @@ def one_ai():
...
     simulator.personPlusAi(cast.KillerOfWorlds)
     #dirname = os.path.dirname(__file__)
-    #script_input(os.path.join(dirname,"scripts/input/"))
+    #script_input(oxs.path.join(dirname,"scripts/input/"))
 
 def two_ai():
     watchTV = setting.InfiniteTelevision()
 
-    simulator.twoAiPlusPerson(cast.Avatar, cast.Unvatar)
+    with open(input("file path to load:")) as f:
```
Comment (emberian) [inline main.py:146]: it'd be cool if this supported a CLI argument in addition to an interactive mode! 

- [ ] abstract this path into an argument to `two_ai`

File: xragents/simulator.py [hunk 3]
```diff
@@ -60,16 +61,12 @@ def personPlusAi(chr: Character):
             audio.cleanup("recording/ai/", "recording/user/") # delete the temporary files
 
         return history
-    
-def twoAiPlusPerson(chr1: Character, chr2: Character):
+
+def interactive_conversation(scene: Scene):
```
Comment (emberian) [inline xragents/simulator.py:65]:
```suggestion
def interactive_conversation(scene: Scene):
    """This is a basic conversation between you and n numbers of AI. Choose your session description and what characters you want."""
```

File: xragents/simulator.py [hunk 4]
```diff
@@ -60,16 +61,12 @@ def personPlusAi(chr: Character):
...
     history = []
-    """This is a basic conversation between you and an AI. Choose your session description and what characters you want."""
-    with scene.make_scene(id=utils.next_session(),
-                    name="Contemplations on Entities",
-                    description=f"The following is an entertaining convo between {chr1.name} and {chr2.name} about the nature of artificial and biological entities, on the substance of souls, individuality, agency, and connection.",
-                    characters=[chr1, chr2],
-                    text_only=True,
-                    ) as sess:
+    # TODO: read a scene object from CSV
```
Comment (emberian) [inline xragents/simulator.py]:
```suggestion
```

File: xragents/simulator.py [hunk 5]
```diff
@@ -60,16 +61,12 @@ def personPlusAi(chr: Character):
...
+    """This is a basic conversation between you and n numbers of AI. Choose your session description and what characters you want."""
```
Comment (emberian) [inline xragents/simulator.py:66]:
```suggestion
```

**Review by emberian — COMMENTED** [review]
left some notes about the commit we made. @Caerii, can you finish up the details here?

File: generate_example_csvs.py (hunk 1 above)
Comment (Caerii) [reply generate_example_csvs.py:27]: I think for cli those flags would be useful, how do we add cli flags as arguments in this situation?

File: main.py (hunk 2 above)
Comment (Caerii) [reply main.py:146]: How do we do such a thing?

**Review by Caerii — APPROVED** [review]

**Pull request merged**